File: C++17/백준/Bronze/1159. 농구 경기/농구 경기.cc
#include<bits/stdc++.h>
using namespace std;
int n,temp;
string s,ret;
vector<int>v;
int arr[26];

int main(){
	ios_base::sync_with_stdio(false);
	cin.tie(0);cout.tie(0);
	
	cin>>n;
	for(int i=0;i<n;i++){
		cin>>s;
		arr[ s[0]-'a']++;
	}
	for(int i=0;i<26;i++){
		if(arr[i]>4){
			ret+= i+'a';
		}
	}
	if(ret.size()) cout<<ret;
	else cout<<"PREDAJA";
	
		
}

File: Java/백준/Bronze/1159. 농구 경기/농구 경기.java

import java.io.*;
import java.util.*;
public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int[]arr= new int[28];
        for (int i = 0; i < n; i++) {
            String s = br.readLine();
            int temp = s.charAt(0) - 'a';
            arr[temp]++;
        }
        String ret="";
//        for (int i : arr) {
        for (int i = 0; i < 26; i++) {
            if(arr[i]>=5) {
                ret += (char) (i + 'a');
            }
        }

        if(ret.length()>0){
            System.out.println(ret);
        }else{
            System.out.println("PREDAJA");
        }
    }
}

